package.json

```diff
@@ -27,12 +27,10 @@
   "devDependencies": {
     "@commitlint/cli": "11.0.0",
     "@commitlint/config-angular": "11.0.0",
-    "@hapi/joi": "17.1.1",
     "@nestjs/common": "7.5.4",
     "@nestjs/core": "7.5.4",
     "@nestjs/platform-express": "7.5.4",
     "@nestjs/testing": "7.5.4",
-    "@types/hapi__joi": "17.1.6",
```

Comment:
You're missing the `@types/joi` package

Reply:
Package `joi` already include typings, or i don't understand something?

Reply:
My bad sorry, we should get [@types/joi](https://www.npmjs.com/package/@types/joi) deprecated then.